app/models/match_decisions/provider_only/confirm_hsa_accepts_client_decline_dnd_staff.rb

```diff
@@ -86,9 +86,9 @@ def decline_confirmed
         # TODO maybe rerun the matching engine for that vacancy and client
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.